roles/setup/README.md

```diff
@@ -0,0 +1,27 @@
+Role Name
+=========
+
+Setup role to be used as a dependency for other roles to conditionally load vaulted 
+variables for setup tasks. 
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+Checks whether the `geniza_deploy_only` variable is defined to determine
```

Comment:
Could we do all this in the `geniza_setup` role? Why is this necessary for testing the postgresql role?

Reply:
Oh, never mind. I am confusing this repo with the princeton_ansible repo . . .